### ISSUE TITLE
Angular 8 compatibility

### ISSUE THREAD

**Issue by mtinnes** [mseemann/angular2-mdl#1746]
The following error surfaces when targeting Angular 8:
```
ERROR in ./node_modules/@angular-mdl/core/components/dialog/mdl-dialog.service.js 247:63-71
"export 'DOCUMENT' was not found in '@angular/platform-browser'
```

According to migration docs the package for DOCUMENT has changed:

> DOCUMENT is removed from @angular/platform-browser If you use DOCUMENT from @angular/platform-browser, you should start to import this from @angular/common.

**Comment by AngularPlayer** [comment]
Hello,  what about Angular 7 compatibilty ?

**Comment by vinay-akkipet** [comment]
We are upgrading to angular 8 and facing the same issue with files like popover.js etc. For now, I have corrected the angular-mdl js files with the correct import statement and placed them separately in a 'patch' folder. Once my build tool runs 'npm install' and before it runs 'ng build'  I have a 'copy' task that copies the corrected files to the correct angular-mdl directory replacing the affected files.

Any helpful suggestions are really appreciated!

**Comment by vinay-akkipet** [comment]
@mtinnes Can you paste your angular.json and package.json? I upgraded to angular 8 recently, I think angular-mdl is compatible with angular 8 if you are using the right version, I see the correct import for the file in question . Here is my package.json dependency declaration for angular-mdl

`"dependencies": {
    "@agm/core": "1.0.0-beta.6",
    "@angular-mdl/core": "^6.0.0",
    "@angular-mdl/datepicker": "6.0.0",
    "@angular-mdl/popover": "^6.0.0",
    "@angular-mdl/select": "^6.0.0",`

**Comment by mseemann** [comment]
https://www.npmjs.com/package/@angular-mdl/core/v/8.0.0-beta.0

**Comment by vinay-akkipet** [comment]
@mseemann any plans for angular-mdl ext?

**Comment by mseemann** [comment]
@vinay-akkipet yes, it will be available in (hopefully) a few days too.

**Comment by mseemann** [comment]
all @angular-mdl packages are now available as version 8.0.0
